### PR TITLE
Use parent instead of base

### PR DESCRIPTION
**Pull request by jjatria**
As per https://github.com/PerlFFI/FFI-Platypus/pull/329, this is to make the equivalent changes here. :D

### CODE REVIEW

File: dist.ini [hunk 1]
```diff
@@ -196,6 +195,7 @@ ExtUtils::MakeMaker = 6.64
 
 [Prereqs / ConfigureRequires]
 -phase = configure
+parent              = 0
```
Comment (plicease) [inline dist.ini:198]: I'm not sure this is required at configure time for this module?

Comment (jjatria) [reply dist.ini:198]: Ah, good catch. I'll change that

**Comment by plicease** [comment]
Some documentation still mentions `base`:

```
✅ doublethink% ack use\ \\bbase\\b lib
lib/Alien/Build/Log.pm
17: use base qw( Alien::Build::Log );

lib/Alien/Build/MM.pm
42: use base qw( Alien::Base );

lib/Alien/Build/Manual/AlienAuthor.pod
490: use base qw( Alien::Base );
528: use base qw( Alien::Base );

lib/Alien/Build/Manual/Contributing.pod
51: use base qw( Alien::Base );

lib/Alien/Base.pm
972: use base qw( Alien::Base );

lib/Alien/Role.pm
14: use base qw( Alien::Base );
```

**Comment by jjatria** [comment]
Ouch, that was sloppily done. I'll push an update tonight with both fixes.

**Comment by plicease** [comment]
> Ouch, that was sloppily done. I'll push an update tonight with both fixes.

Nah, just the review process working :)

**Comment by jjatria** [comment]
I've made the requested changes.

Code review++ !

**Pull request merged**